node_mongo_tutorial/routes.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const Post = require("./models/Post");
+const router = express.Router();
+
+// Get all posts
+router.get("/posts", async (req, res) => {
+  const posts = await Post.find();
+  res.send(posts);
+});
+
+// Get Single Post
+router.get("/posts/:id", async (req, res) => {
+  try {
+    const post = await Post.findOne({ _id: req.params.id });
+    res.send(post);
+  } catch {
+    res.status(404);
```

Comment:
You need to wrap res.status with res.json
and it should looks like this 
res.status(200).json({
 status: 200,
 message: "Message here",
 data: {}
})

Reply:
I'm sorry but this was what I learnt from the tutorial.... but I have corrected it in the project... you can check out #20